apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.tsx

```diff
@@ -74,9 +79,10 @@ function LaunchWpcomWelcomeTour() {
 		siteIntent,
 		siteIntentFetched,
 		editorType,
+		showLaunchpad,
 	] );
 
-	if ( ! show || isNewPageLayoutModalOpen ) {
+	if ( ! show || isNewPageLayoutModalOpen || showLaunchpad ) {
```

Comment:
Hey  @arthur791004, the 'showLaunchpad' attribute is only used on the new start-writing flow, so this will not affect other flows. Btw, we planned to change the name of the attribute in the near future

Reply:
No, the `showLaunchpad` is used for a long time in [the site-setup flow](https://github.com/Automattic/wp-calypso/blob/5a6db6d2a8241300d8bd1921af5c7d7418c8b779/client/landing/stepper/declarative-flow/site-setup-flow.ts#L231). This will affect the user going through the pattern assembler screen and land on the site editor directly as the Welcome Tour will be not available at that time.

> Btw, we planned to change the name of the attribute in the near future

This sounds better but it has already had a side effect now 😓

Reply:
We didn't know about that, we will see it tomorrow morning to avoid the side effects

Reply:
@arthur791004, I will prioritize these changes, and will be published ASAP.